test/utils/date_time_extension_test.dart

```diff
@@ -0,0 +1,337 @@
+import 'package:fluffychat/utils/date_time_extension.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('[localizedTimeShort]', () {
+    testWidgets(
+        'should return time of day with format hh:mm when date is the same day',
+        (WidgetTester tester) async {
+      await tester.binding.setLocale('en', 'US');
+      await tester.pumpWidget(
+        MaterialApp(
+          localizationsDelegates: const [L10n.delegate],
+          home: Builder(
+            builder: (BuildContext context) {
+              final currentTime = DateTime(2022, 1, 1);
+              final timeToTest = DateTime(2022, 1, 1, 12, 5);
+
+              const expectedResult = '12:05';
+
+              final result = timeToTest.localizedTimeShort(
+                context,
+                currentTime: currentTime,
+              );
+
+              expect(result, equals(expectedResult));
```

Comment:
IMO: SHOULD NOT use `expect` in widget. Will be app the docs for widget test If you want 

docs: https://docs.flutter.dev/cookbook/testing/widget/introduction